medical_medicament_us/demo/medical_medicament_gcn_demo.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 LasLabs Inc.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record id="medical_medicament_gcn_us_1" model="medical.medicament.gcn">
+        <field name="name">Demo GCN</field>
```

Comment:
ADVIL 200 MG TABLET & CAPLET with NDC of:
* 00573015046
* 00573016030
* 00573016020
* 00573015175
* 00573015176
* 00573016043
* 00573016040
* 00573015008

Has a GCN 8346

This is the same GCN as Ibuprofen 200, Tylenol 200, and anything else that has a formulary of 200mg of Ibuprofen.